package.json

```diff
@@ -86,33 +86,34 @@
 		"watch": "run-p apollo:watch prisma:watch"
 	},
 	"dependencies": {
-		"@apollo/client": "3.5.8",
-		"@chakra-ui/icons": "1.1.1",
-		"@chakra-ui/react": "1.8.1",
-		"@chakra-ui/system": "1.10.1",
-		"@emotion/cache": "11.7.1",
-		"@emotion/css": "11.7.1",
-		"@emotion/react": "11.7.1",
-		"@emotion/server": "11.4.0",
-		"@emotion/styled": "11.6.0",
-		"@prisma/client": "3.8.1",
-		"apollo-server-micro": "3.6.2",
-		"framer-motion": "5.6.0",
-		"graphql": "15.8.0",
-		"graphql-scalars": "1.14.1",
-		"iron-session": "6.0.5",
-		"lodash": "4.17.21",
-		"micro": "9.3.4",
-		"micro-cors": "0.1.1",
-		"next": "12.0.9",
-		"nexus": "1.2.0",
-		"react": "17.0.2",
-		"react-dom": "17.0.2",
-		"react-hook-form": "7.25.3",
-		"react-icons": "4.3.1",
-		"react-table": "7.7.0",
-		"sharp": "0.29.3",
-		"yup": "0.32.11"
+		"@apollo/client": "^3.5.8",
+		"@chakra-ui/icons": "^1.1.1",
+		"@chakra-ui/react": "^1.8.1",
+		"@chakra-ui/system": "^1.10.1",
+		"@emotion/cache": "^11.7.1",
+		"@emotion/css": "^11.7.1",
+		"@emotion/react": "^11.7.1",
+		"@emotion/server": "^11.4.0",
+		"@emotion/styled": "^11.6.0",
+		"@prisma/client": "^3.8.1",
+		"apollo-server-micro": "^3.6.2",
+		"argon2": "^0.28.3",
+		"framer-motion": "^5.x.x",
+		"graphql": "^15.x.x",
+		"graphql-scalars": "^1.14.1",
+		"iron-session": "^6.0.5",
+		"lodash": "^4.17.21",
+		"micro": "^9.3.4",
+		"micro-cors": "^0.1.1",
+		"next": "^12.0.9",
+		"nexus": "^1.2.0",
+		"react": "^17.0.2",
+		"react-dom": "^17.0.2",
+		"react-hook-form": "^7.25.1",
+		"react-icons": "^4.3.1",
+		"react-table": "^7.7.0",
+		"sharp": "^0.29.3",
+		"yup": "^0.32.11"
```

Comment:
Something messed up merging here, all dependencies have to stay pinned for renovate.